README.md

```diff
@@ -3,14 +3,19 @@ GPIO
 
 Read and write on a variety of gpio interfaces.
 
+Dependencies
+-----------
+* [RPi.GPIO](https://pypi.python.org/pypi/RPi.GPIO)
+
 GPIORead
 ========
 
-Read the bool value of a gpio pin.
+Read the value of a gpio pin.
 
 Properties
 ----------
-- Pin (bool) - ping to read from.
+- **pin** (int): pin to read from
+- **pull_up_down**: direct `pin` to known state
```

Comment:
I think you've got this right but the phrasing isn't clear to me. Maybe something like, "Value of pin when it's logic level is neither high nor low"

https://en.wikipedia.org/wiki/Pull-up_resistor